site/_data/cli/pulsar-admin.yaml

```diff
@@ -510,6 +510,164 @@ commands:
     argument: cluster-name
   - name: delete
     description: Delete namespace isolation policy of a cluster. This operation requires superuser privileges.
+- name: sink
+  description: An interface for managing Pulsar IO sinks (egree data from Pulsar)
+  subcommands:
+  - name: create
+    description: Create a Pulsar IO sink connector
```

Comment:
Here and elsewhere, can we add something that says that this connector will start running in the pulsar cluster.